### ISSUE TITLE
trying to use fortigate image in clabernetes

### ISSUE THREAD

**Issue by HaroldYE** [srl-labs/clabernetes#168]
fortgate image is not supported in Clabernetes

INFO |              containerlab | Error: error constructing node "f5": kind "fortinet_fortigate" is not supported. Supported kinds are "arista_ceos, arista_veos, aruba_aoscx, border0, bridge, c8000, ceos, checkpoint_cloudguard, cisco_c8000, cisco_csr1000v, cisco_ftdv, cisco_n9kv, cisco_xrd, cisco_xrv, cisco_xrv9k, crpd, cumulus_cvx, cvx, dell_ftosv, ext-container, host, ipinfusion_ocnos, juniper_crpd, juniper_vjunosevolved, juniper_vjunosswitch, juniper_vmx, juniper_vqfx, juniper_vsrx, k8s-kind, keysight_ixia-c-one, linux, mikrotik_ros, nokia_srlinux, nokia_sros, openbsd, ovs-bridge, paloalto_panos, rare, sonic-vs, srl, vr-aoscx, vr-arista_veos, vr-aruba_aoscx, vr-cisco_csr1000v, vr-cisco_n9kv, vr-cisco_xrv, vr-cisco_xrv9k, vr-csr, vr-dell_ftosv, vr-ftosv, vr-juniper_vmx, vr-juniper_vqfx, vr-juniper_vsrx, vr-mikrotik_ros, vr-n9kv, vr-nokia_sros, vr-paloalto_panos, vr-pan, vr-ros, vr-sros, vr-veos, vr-vmx, vr-vqfx, vr-vsrx, vr-xrv, vr-xrv9k, xrd"

CRITICAL |               clabernetes | failed launching containerlab, will try to gather crashed container logs then will exit, err: exit status 1

**Comment by hellt** [comment]
yes, at this moment clabernetes runs containerlab 0.51.3, as we have some unresolved vxlan issues with newer versions, and fortigate was added in 0.53.0

**Comment by carlmontanari** [comment]
resolved in a round about way from clab 2213 and c9s #178 -- will be supported in next c9s release.
